src/fmeval/eval_algorithms/qa_accuracy.py

```diff
@@ -105,6 +105,17 @@ def _normalize_and_strip_text(text: str, *, normalize_text: bool = False, strip_
     return text
 
 
+def _split(text: str) -> List[str]:
+    """
+    Split text matching one or more spaces (\s+) or one or more new lines (\n+) or any other string.whitespace
```

Comment:
For the sake of conciseness, consider updating this to: Split text based on `string.whitespace` characters.
Reason: space and newline are included in `string.whitespace` characters.